checks/ci_tests.go

```diff
@@ -39,17 +39,17 @@ func init() {
 
 // CITests runs CI-Tests check.
 func CITests(c *checker.CheckRequest) checker.CheckResult {
-	prs, err := c.RepoClient.ListMergedPRs()
+	commits, err := c.RepoClient.ListCommits()
 	if err != nil {
-		e := sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("RepoClient.ListMergedPRs: %v", err))
+		e := sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("RepoClient.ListCommits: %v", err))
 		return checker.CreateRuntimeErrorResult(CheckCITests, e)
 	}
 
 	totalMerged := 0
 	totalTested := 0
-	for index := range prs {
-		pr := &prs[index]
-		if pr.MergedAt.IsZero() {
+	for i := range commits {
+		pr := &commits[i].AssociatedMergeRequest
+		if pr.MergedAt.IsZero() || pr.Repository != c.Repo.String() {
```

Comment:
I'm curious: does ListCommits() list all commits to branches used during PRs? If not, why is `pr.Repository != c.Repo.String()` needed?

Reply:
`ListCommits` is basically `git log` on a repository. However, the associated PR with that commit might come from a different repo (say in case of a fork). So, I'm ignoring such PRs to maintain our current behavior. 

There is a bigger discussion to be had on whether we should honor commits whose PRs originated on a different repo. i.e, should such commits be considered code reviewed etc. But I don't want that discussion to affect this PR. The aim of this PR is to remove the API but keep behavior the same.

Reply:
> `ListCommits` is basically `git log` on a repository. However, the associated PR with that commit might come from a different repo (say in case of a fork). So, I'm ignoring such PRs to maintain our current behavior.

sorry, what does a different repo mean? When we submit PR to scorecard repo, we use a different repo, which is a fork of the original one. But it is technically a different repo, right? Does it mean these PRs are ignored? Or does "different repo" mean something different?

> There is a bigger discussion to be had on whether we should honor commits whose PRs originated on a different repo. i.e, should such commits be considered code reviewed etc. But I don't want that discussion to affect this PR. The aim of this PR is to remove the API but keep behavior the same.

Reply:
So let's say you run Scorecard on `laurentsimon/scorecard`. `ListCommits()` will return [58865e959e3a782d3f3cd5a5ae952ac308c11a46](https://github.com/laurentsimon/scorecard/commit/58865e959e3a782d3f3cd5a5ae952ac308c11a46)  as one of the commits in this repository. But the PR which created this commit is https://github.com/ossf/scorecard/pull/1562.  

So `commit.associatedPR` will point to #1562 which belongs to a different repository. Does that make sense?

Reply:
ho I see. So this only affects forks. Let's add a comment to the code to clarify things. I suppose it's OK to consider these as reviews, but we will discuss this in another issue. Thanks for the explaination

Reply:
Done.